aspnetcore/fundamentals/websockets.md

```diff
@@ -88,8 +88,16 @@ When accepting the WebSocket connection before beginning the loop, the middlewar
 
 Windows Server 2012 or later and Windows 8 or later with IIS/IIS Express 8 or later has support for the WebSocket protocol.
 
+> [!NOTE]
+> WebSockets are always enabled when using IIS Express.
+
+### Enabling WebSockets on IIS
+
 To enable support for the WebSocket protocol on Windows Server 2012 or later:
 
+> [!NOTE]
+> These steps are not required when using IIS Express
```

Comment:
`>Note` chrome is really distracting. I think this section would be much better if you replaced

## IIS/IIS Express support

with 

## IIS Express support

WebSockets are enabled when using IIS Express.

## IIS support

That way you can get rid of all those distracting notes These steps are not required when using IIS Express

Reply:
That would work, though there would be some repetition of text under the two headings.

Reply:
@tdykstra 	what needs to be duplicated?

Reply:
Some of this::  "Windows Server 2012 or later and Windows 8 or later with IIS/IIS Express 8 or later has support for the WebSocket protocol."